### ISSUE TITLE
Intermittent failure in RunsInThreadsTest.matchNegative

### ISSUE THREAD

**Issue by llorllale** [llorllale/cactoos-matchers#51]
[Log](https://travis-ci.org/llorllale/cactoos-matchers/builds/457480519?utm_source=github_status&utm_medium=notification)

```stacktrace
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.168 sec <<< FAILURE! - in org.llorllale.cactoos.matchers.RunsInThreadsTest
matchNegative(org.llorllale.cactoos.matchers.RunsInThreadsTest)  Time elapsed: 0.167 sec  <<< FAILURE!
java.lang.AssertionError: 
Can concurrently modify HashMap
Expected: <false>
     but: was <true>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.llorllale.cactoos.matchers.RunsInThreadsTest.matchNegative(RunsInThreadsTest.java:61)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:393)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by llorllale** [comment]
@0crat in

**Comment by 0crat** [comment]
Job [#51](https://github.com/llorllale/cactoos-matchers/issues/51) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/llorllale/cactoos-matchers/issues/51#issuecomment-445488885))

@llorllale Job [#51](https://github.com/llorllale/cactoos-matchers/issues/51) is already in scope

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@dgroup does this still happen to you?

**Comment by 0crat** [comment]
The job [#51](https://github.com/llorllale/cactoos-matchers/issues/51) assigned to @verkhovin[/z](https://www.0crat.com/u/verkhovin), here is [why](https://www.0crat.com/footprint/C63314D6Z/2df43dae-6a61-4803-8cfb-4aaacf851262); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by verkhovin** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/llorllale/cactoos-matchers/issues/51#issuecomment-453786777))

@verkhovin The user @verkhovin[/z](https://www.0crat.com/u/verkhovin) resigned from [#51](https://github.com/llorllale/cactoos-matchers/issues/51), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @verkhovin[/z](https://www.0crat.com/u/verkhovin)

**Comment by llorllale** [comment]
@0crat assign me

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/llorllale/cactoos-matchers/issues/51#issuecomment-454224631))

@llorllale The job [#51](https://github.com/llorllale/cactoos-matchers/issues/51) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C63314D6Z/7f0852e3-e01d-4afe-8f96-d69632d6c9fc); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); we should be aware that @llorllale[/z](https://www.0crat.com/u/llorllale) is on vacation; this ticket may be delayed; there will be no monetary reward for this job

**Comment by 0crat** [comment]
It is strongly discouraged to assign jobs to their creators, see [§19](http://www.zerocracy.com/policy.html#19): -15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@0crat wait because I submitted PR #77 for this

**Comment by 0crat** [comment]
> @0crat wait because I submitted PR #77 for this ([here](https://github.com/llorllale/cactoos-matchers/issues/51#issuecomment-454247363))

@llorllale The impediment for [#51](https://github.com/llorllale/cactoos-matchers/issues/51) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@llorlale fixed in PR #77

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)
